### DISCUSSION TITLE
Example of using dir-locals

### DISCUSSION THREAD

**Discussion by edmundmiller** [emacs-citar/citar#748] [Q&A] [answered]
Just wondering if there's an example for using `.dir-locals.el` for `citar-bibliography` for a writing project. Something like

```elisp
((nil . ((citar-bibliography . "~/path/to/project/citations.bib"))))
```
Seems to be working. I guess I'm just wondering if there's a better way to handle this?

**Comment by bdarcus** [comment]
I moved this over to discussions @Emiller88.

Oh, I hadn't even thought of that!

I guess the details would depend on what major-mode you're using. Might not work as well with org, I'd guess?

And you probably can't see the citation style for the document?

Are you using `LaTeX-mode`?

**Reply by edmundmiller** [reply]
I'm using org. I actually wrapped up writing earlier this week for now, but I was just playing around with it and it seemed to work!

**Reply by edmundmiller** [reply, marked as answer]
```elisp
((org-mode . ((org-attach-id-dir . "~/src/Dissertation/org-attach/")
              (citar-bibliography . "~/src/Dissertation/qe/citations.bib")
              (org-cite-global-bibliography . ("~/src/Dissertation/qe/citations.bib")))))
```
🤯
